src/components/popover/popover.test.jsx

```diff
@@ -106,7 +113,26 @@ describe('Popover', () => {
     expect(screen.getByText('Another title').tagName).toEqual('H1');
   });
 
-  describe('flush api', () => {
+  describe('autoflush API', () => {
+    // We don't know how to test this well. Manually QA for now.
+  });
```

Comment:
TODO for me: I want to find a way to test this, I know there is one, I should come back to it

I do like that you added _something_ here for it, I feel like that points out that we intentionally don't have test coverage here instead of making it look like its something we just missed.